package.json

```diff
@@ -126,7 +126,7 @@
     "jest-canvas-mock": "^2.3.1",
     "jest-puppeteer": "^6.0.0",
     "postcss": "^8.3.0",
-    "prettier": "^2.2.1",
+    "prettier": "^2.5.1",
```

Comment:
Why are you changing the version of prettier?

Reply:
I had installed it manually, I just changed the prettier version to the original one.